HULKs/hulk
# Use alternative static ball model

## Introduced Changes

We noticed that the robot can shoot significantly better when the ball model does not jitter.
This PR adds a resting equivalent to each hypothesis, which has higher velocity measurement noise and therefore is slower and doesn't have as much position noise.
The switch between moving and resting ball is determined depending on the velocity of the moving ball.


Fixes #

## ToDo / Known Issues

*If this is a WIP describe which problems are to be fixed.*

## Ideas for Next Iterations (Not This PR)

*If there are some improvements that could be done in a next iteration, describe them here.*

## How to Test

* Observe the `chooses_resting_model` additional output to see when the robot switches between the models.
* The robot should make a kick decision faster when the ball is steady (given the robot is properly localized)
